libs/eavmlib/src/avm_pubsub.erl

```diff
@@ -0,0 +1,107 @@
+-module(avm_pubsub).
+
+-export([start/0, start/1, pub/3, sub/2, sub/3, unsub/2, unsub/3]).
+-export([init/1, handle_call/3, handle_info/2, terminate/2]).
+
+start() ->
+    gen_server:start(?MODULE, [], []).
+
+start(LocalName) ->
+    gen_server:start({local, LocalName}, ?MODULE, [], []).
+
+pub(PubSub, Topic, Term) ->
+    gen_server:call(PubSub, {pub, Topic, Term}).
+
+sub(PubSub, Topic) ->
+    gen_server:call(PubSub, {sub, Topic}).
+
+sub(PubSub, Topic, Pid) ->
+    gen_server:call(PubSub, {sub, Topic, Pid}).
+
+unsub(PubSub, Topic) ->
+    gen_server:call(PubSub, {unsub, Topic}).
+
+unsub(PubSub, Topic, Pid) ->
+    gen_server:call(PubSub, {unsub, Topic, Pid}).
+
+init(_) ->
+    {ok, []}.
+
+handle_call({pub, Topic, Term}, {FromPid, _Ref}, Table) ->
```

Comment:
Does this need to be a call?  Or could it be a cast?  Or, do you want a call version and a cast version of the API?

Reply:
I wanted to return the number of subscribers that received a publish, so that was the way. I just wanted to draft a minimal API, we always have time to add more functions.